### PR TITLE
sylpheed: update to 3.7.0

### PR DESCRIPTION
**Pull request by abenson**
Also added LDAP support to build, as requested by stateless on IRC.

### CODE REVIEW

**Comment by cr6git** [comment]
it closes [#11370](https://github.com/voidlinux/void-packages/pull/11370) then.

File: srcpkgs/sylpheed/template [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
 short_desc="GTK+ Lightweight and user-friendly e-mail client"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2, LGPL-2.1"
 homepage="http://sylpheed.sraoss.jp"
 distfiles="http://sylpheed.sraoss.jp/sylpheed/v${version%.*}/$pkgname-$version.tar.bz2"
-checksum=eba4a70b43b49dbf419934796677a69a71d069b71b0dd697a0008836be4db345
+checksum=eb23e6bda2c02095dfb0130668cf7c75d1f256904e3a7337815b4da5cb72eb04
+
+if [ -n "$CROSS_BUILD" ]; then
+	hostmakedepends+=" gettext-devel gtk+-devel gpgme-devel glib-devel"
```
Comment (Vaelatern) [inline srcpkgs/sylpheed/template]: comment explaining why this conditional?

Comment (abenson) [reply srcpkgs/sylpheed/template]: It seems to only be needed when cross compiling. I'll try and work it to be cleaner

Comment (Vaelatern) [reply srcpkgs/sylpheed/template]: What's wrong with unconditionally keeping it?

**Comment by abenson** [comment]
I changed it back to just the changes necessary.

**Pull request merged**